src/three-components/SmoothControls.ts

```diff
@@ -541,6 +590,11 @@ export class SmoothControls extends EventDispatcher {
   private[$handlePointerUp](_event: MouseEvent|TouchEvent) {
     this.element.style.cursor = 'grab';
     this[$pointerIsDown] = false;
+    this.setOrbit(
+      this[$clampedTargetSpherical].theta,
+      this[$clampedTargetSpherical].phi,
+      this[$clampedTargetSpherical].radius
```

Comment:
Basically when the user "lets go" we just want to orbit back to the clamped target.